server/routes/teams.ts

```diff
@@ -46,6 +46,11 @@ export default function routes({
     const teamSlug = getFormattedName(req, 'teamSlug')
     const team = await serviceCatalogueService.getTeam({ teamSlug, withEnvironments: true })
     const products = team.products.map(product => product)
+    const components = getComponentsForTeam(team)
+    let componentList: string[] = []
```

Comment:
`const componentList: string[] = (components && components.length > 0) ? components.map(component => component.componentName) : []`

Gets rid of the `let` and the `if`